### DISCUSSION TITLE
Welcome to PWA Boilerplate Discussions!

### DISCUSSION THREAD

**Discussion by Tomburgs** [Tomburgs/pwa-boilerplate#136] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tomburgs** [comment]
🍻 🍻 🍻

**Reply by morrisonak** [reply]
Great project so far.    Looking forward to more documentation posted on how to further build it out.   I'm planning on using it for foreign language learning app.  
Thanks.
